config.go

```diff
@@ -805,20 +805,27 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 		if conf.RPCUser != "" && conf.RPCPass != "" {
 			return nil
 		}
+
+		// Get the daemon name for displaying proper errors
+		switch net {
+		case bitcoinChain:
+			daemonName = "btcd"
+		case litecoinChain:
+			daemonName = "ltcd"
+		}
+
 		// If only ONE of RPCUser or RPCPass is set, we assume the
 		// user did that unintentionally.
 		if conf.RPCUser != "" || conf.RPCPass != "" {
 			return fmt.Errorf("please set both or neither of " +
-				"btcd.rpcuser and btcd.rpcpass")
+				daemonName + ".rpcuser and " + daemonName + ".rpcpass")
```

Comment:
Since we’re already using `Errorf`, this can use formatting directives instead of string concatenation

Reply:
I'm not super-familiar with go, going to update the commit like this - 

```
                        return fmt.Errorf("please set all or none of " +
                                "%[1]v.rpcuser, %[1]v.rpcpass, " +
                                "and %[1]v.zmqpath", daemonName)
```

Does that look correct?